### PR TITLE
openfaas chart

### PR DESCRIPTION
**Pull request by rberrelleza**
Still needs to be tested end to end, but getting the PR going.

It's openfaas 5.4 with some modifications so that RBAC is not created.  I'll send  the PR to openfaas once we have it running here.

### CODE REVIEW

**Comment by rberrelleza** [comment]
After deploying this, you have 6GB of Ram and 2.2 CPUs left for functions and dev envs. @pchico83 what do you think?  (consider that long term we either enable namespace to namespace communication or we have a multi-tenant OpenFaas install).

File: openfaas/config.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# this will be replaced by Okteto during deployment
+functionNamespace: $NAMESPACE
+serviceType: ClusterIP
+
+# okteto manages the RBAC 
+rbac: false
+
+# disabled by default to make development easier
+basic_auth: false
```
Comment (pchico83) [inline openfaas/config.yaml]: Would it be possible to enable it and configure the password here, in the config.yaml, with a default value?

File: openfaas/config.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+alertmanager:
+    limits:
+      memory: "50Mi"
+      cpu: "100m"
+
+nats:
+  resources:
+    limits:
+      memory: "120Mi"
+    requests:
```
Comment (pchico83) [inline openfaas/config.yaml]: Set just the `limits`, Okteto Cloud sets `requests` to zero.
Also, setting `limits.cpu` we can reduce the total cpu needed

**Comment by pchico83** [comment]
@rberrelleza do you know how to configure the auto-scale to zero? can we show the option at the config.yaml file?

File: openfaas/config.yaml (hunk 1 above)
Comment (rberrelleza) [reply openfaas/config.yaml]: we can configure it to have auth enabled, and then the chart creates the secret. but then the user needs to go to the CLI to get the secret before being able to use the gateway. I fee like it's a lot of friction. what do you think @pchico83 ?

**Comment by rberrelleza** [comment]
> @rberrelleza do you know how to configure the auto-scale to zero? can we show the option at the config.yaml file?

no, but let me test it out, it's an scenario that would be cool to support

File: openfaas/config.yaml (hunk 1 above)
Comment (pchico83) [reply openfaas/config.yaml]: I think the openfaas gateway shouldnt be open to the world... I think that if the user configures the password in the config-yaml that the Okteto Cloud UI shows (with a default value) is not too much friction

**Comment by pchico83** [comment]
> > @rberrelleza do you know how to configure the auto-scale to zero? can we show the option at the config.yaml file?
> 
> no, but let me test it out, it's an scenario that would be cool to support

Yeap. A user was already asking for it in the #okteto channel

**Comment by rberrelleza** [comment]
I ended up making some changes on `values.yaml` for the following cases to that the deployer can't change rbac and serviceType via the okteto ui, since it would break the deployment.

I also updated the install secret so that the user can pass her own password, instead of autogenerate.

**Pull request merged**